### ISSUE TITLE
mh

### ISSUE THREAD

**Issue by iamlambda** [Shehroz-Ahmad12/TicTacToe-Python#1]
bro, using letters for defining your board is a very bad idea, you end up with 1 thousand line of code for such a simple program...

**Comment by Shehroz-Ahmad12** [comment]
Thank you for your concern.
That was one of the first codes I had written in programming language in my
first semester. And as you can see most of the code is repetitive and does
not follow any standards.
That's why rookie mistakes i guess xD

**Comment by iamlambda** [comment]
yeah right, we could try to make a project together, would you be down? if yes, add me on discord: darling#7177 
(:

**Comment by Shehroz-Ahmad12** [comment]
Sure..
